workflows/iuf/stages.yaml

```diff
@@ -41,6 +41,8 @@ stages:
   - name: deliver-product
     type: product
     operations:
+      - name: add-product-to-product-catalog
```

Comment:
I'm not sure, but cray-nls might require the `local-path` field be present even though it's not used. So if you see an issue in testing, this might be why.

I did open a Jira today to remove this unused field entirely. See [CASMINST-5689](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5689)

Reply:
With the current version on frigg, it was not required, I was able to successfully test this workflow template as written. Checking to see if there is still some reason that I might want to add it.

Reply:
We ended up talking this over in slack and decided to leave this as-is. We think this might not work with some versions of cray-nls in which the required fields are validated (and also in which local-path is a required field), but @atifsyedali is planning on removing local-path as a field entirely before the next deployment anyway.